### ISSUE TITLE
Missing module in tarball

### ISSUE THREAD

**Issue by snoyberg** [zohl/servant-auth-cookie#42]
To repro: try using `stack unpack` or `cabal unpack` to see the files actually on Hackage.

```
[1 of 1] Compiling Main             ( tests/Main.hs, dist/build/tests/tests-tmp/Main.o )

tests/Main.hs:21:1: error:
    Could not find module ‘Utils’
    Use -v to see a list of the files searched for.
   |
21 | import Utils
   | ^^^^^^^^^^^^
```

**Comment by zohl** [comment]
Hello,

Thank you for noticing! I fixed this in version `0.6.0.1`.

**Comment by snoyberg** [comment]
Thanks!
